MukurtuCMS/Mukurtu-CMS
# Eckmo - theme Dashboard

- Added logo to theme info.yml (this must be done for png logos as Drupal looks for svg)
- Add templates for Dashboard
- Add styling for Dashboard 
- Tweak globals based on contextual needs
- Set up site grid 
- Set up page template and extend the page template in the Dashboard template
- Style account menu
- Style header 
- First pass at main nav (this will likely change a bit with accessibility improvements)
- Breadcrumbs including current page displaying in the path 

**As it turns out, Drupal doesn't support svg logos out of the box, but the png is just fine :-) 

**This now includes all of original theme scaffolding changes, as well as anything dashboard/breadcrumbs/menu-related. 

**To test this:**
- From a fresh site install, pull this branch locally
- Log into the site 
- From within the `mukurtu_v4` directory, run `npm install` and then `gulp sass` (note, you will have to have `sass-cli` installed globally on your machine for `gulp sass` to work)
- Clear the cache
- Verify that the styles have applied (if not, I may have missed a step in my above instructions so let me know and we can troubleshoot!) 
- _Note, there were a few block titles that I wound up deselecting the 'show title' option on; I couldn't remember which ones they were to pass along the specific config items, but the title for one was "Mukurtu" and the other was "Dashboard"_
- This addresses basic Dashboard styling (I'm sure this will change as content is added -I know each section will eventually be an accordion but this didn't feel possible with the content there currently)
- Account menu styling
- Some main menu styling (expected to change with accessibility improvements, though should look more or less the same)
- Breadcrumb creation and styling 

**There is also the preprocess_hook in the .theme file now - @steve-taylor-wsu I know you mentioned `hook_system_breadcrumb_alter`, and I hope this respects that but if not let me know and maybe we can get together to fix that?
